src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
+  } else if (days >= 3) {
+    finalPrice = 40 * days - 20;
+  } else {
+    finalPrice = 40 * days;
+  }
```

Comment:
Refactor the conditional statements to use `if` with `return` to simplify the logic and avoid unnecessary `else` blocks.